### PR TITLE
i18n: update doc installation spanish translations

### PR DESCRIPTION
**Pull request by Kaian**
Spanish translation for de Install section of documentation.

This PR is part of #723

### CODE REVIEW

File: doc/sphinx/locale/es/LC_MESSAGES/basic_concepts.po [hunk 1]
```diff
@@ -250,110 +250,143 @@ msgstr ""
...
+"caso de utilizar una única dirección IP, los puertos SIP del proxy de "
+"salida se cambiarán para evitar la colisión entre ambos."
 
 #: ../../basic_concepts/installation/cd_install.rst:55
 msgid ""
 "You can set its addresses right now and configure the interfaces properly"
 " when the system is fully installed. This menu can be displayed anytime "
 "after the installation."
 msgstr ""
+"Puede asignar sus valores ahora y configurar sus interfaces mas tarde, o "
```
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/basic_concepts.po]: más

File: doc/sphinx/locale/es/LC_MESSAGES/basic_concepts.po [hunk 2]
```diff
@@ -250,110 +250,143 @@ msgstr ""
...
-msgstr ""
+msgstr "Y el idioma por defecto de los portales web:"
 
 #: ../../basic_concepts/installation/cd_install.rst:72
 msgid ""
 "It is not require to configure all settings during initial installation. "
 "In case any setting has been left without configuration a warning dialog "
 "will be displayed."
 msgstr ""
+"No es preciso configurar todas estas cosas durante la instalación.En caso "
```
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/basic_concepts.po]: Missing whitespace after the dot

File: doc/sphinx/locale/es/LC_MESSAGES/basic_concepts.po [hunk 3]
```diff
@@ -453,10 +506,14 @@ msgid ""
 "<https://github.com/irontec/ivozprovider/issues/271>`_ for more "
 "information."
 msgstr ""
+"Las instalaciones distribuidas requieren multiples configuraciones en "
+"funcion del rol que se haya instalado. Tenga encuenta que este proceso de "
```
Comment (mmadariaga) [inline doc/sphinx/locale/es/LC_MESSAGES/basic_concepts.po]: en cuenta

**Comment by Kaian** [comment]
Updated with requested changes

**Pull request merged**